CS.Security/Helpers/DataSeeder.cs

```diff
@@ -21,24 +22,32 @@ public async Task SeedAllData()
     
     public async Task SeedRoles() 
     {
-        if (!await _roleManager.RoleExistsAsync("User"))
-            await _roleManager.CreateAsync(new IdentityRole<Guid>("User"));
-        if (!await _roleManager.RoleExistsAsync("Admin"))
-            await _roleManager.CreateAsync(new IdentityRole<Guid>("Admin"));
-        if (!await _roleManager.RoleExistsAsync("SuperAdmin"))
-            await _roleManager.CreateAsync(new IdentityRole<Guid>("SuperAdmin"));
+        if (!await _roleManager.RoleExistsAsync(UserRoles.User))
+        {
```

Comment:
It depends on every project's code convention, but in a vast majority of cases it's a rule to cover lines like that with curly braces